game/scripts/vscripts/Sections/Chapter4/SectionWards.ts

```diff
@@ -0,0 +1,178 @@
+import * as tg from "../../TutorialGraph/index";
+import * as tut from "../../Tutorial/Core";
+import { getOrError, getPlayerHero, printEventTable } from "../../util";
+import { RequiredState } from "../../Tutorial/RequiredState";
+import { TutorialContext } from "../../TutorialGraph/index";
+
+enum NeutralGoalKeys {
```

Comment:
Not that it really matters, but maybe it should be named differently? Something like `Chapter4WardGoalKeys`?